### ISSUE TITLE
Not sure if this is an issue for this build, but definatly worth looking into

### ISSUE THREAD

**Issue by GavinDarkglider** [lakka-switch/Lakka-LibreELEC#65]
First I would like to say thank you, because without the work done here, my gentoo l4t build would have never made it to a working state. That being said, following your implementation of the video card driver setup, I noticed that GLES is broken, and you did it all wrong. The way it is done here gets opengl working only. In order to have the whole stack working, you need mesa and glvnd. look here for details on a working implementation(I am still working on openmax, cuda, and tensorRT, hince them not being included, but it does have a full working graphics stack otherwise): https://github.com/GavinDarkglider/gentoo-switch_overlay/blob/master/x11-drivers/jetson-tx1-drivers/jetson-tx1-drivers-32.1.0.ebuild

I wouldnt make this a priority since opengl works, but having a working gles option might improve battery life a little.

**Comment by natinusala** [comment]
Why would you need mesa, doesn't the L4T drivers package include libGLESv2 as well?

But anyway, I don't care about GLES, cores and RA cannot be built for both at the same time (compile time defines), and I don't want to provide multiple builds for that

**Comment by GavinDarkglider** [comment]
Because the gles stuff works via glvnd, which is tied into mesa. OpenGL also works through glvnd, but will work without it as well. I thought I would bring it up in case you were interested, and incase anyone else that was planning on porting another version of linux to the switch, if they looked here for info on how the drivers are installed, they wouldnt run into the issues I ran into figuring it out. Since a lot of my work there has been writing ebuilds based upon what I learned from this github, and figured out on my own. The kernel stuff was quite helpful. ;)
